specification/security/resource-manager/Microsoft.Security/preview/2023-01-01-preview/examples/GetDataScanner_example.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2023-01-01-preview",
```

Comment:
The actual API that was added is `2021-12-01-preview` - and it has an 'identity' property bag - I don't see any properties for the dataScanner resource - was that on purpose?

Reply:
yes.
the main goal of this resource is having a System Managed identity attached to it. at the moment it doesn't have any properties, so we don't have a property bag.

this is similar to this one :
https://learn.microsoft.com/en-us/rest/api/defenderforcloud/security-operators/get?view=rest-defenderforcloud-2023-01-01-preview&tabs=HTTP#securityoperator

which we introduced after the dataScanners